tools/internal_ci/linux/arm64/grpc_bazel_test_c_cpp_in_docker.sh

```diff
@@ -25,5 +25,13 @@ cd /var/local/git/grpc
 # tests require port server to be running
 python3 tools/run_tests/start_port_server.py
 
+python3 tools/run_tests/python_utils/bazel_report_helper.py --report_path bazel_test_c_cpp
+
 # test gRPC C/C++ with bazel
-tools/bazel test --config=opt --test_output=errors --test_tag_filters=-no_linux,-no_arm64 --build_tag_filters=-no_linux,-no_arm64 --flaky_test_attempts=1 --runs_per_test=1 //test/...
+bazel_test_c_cpp/bazel_wrapper \
+  --bazelrc=tools/remote_build/include/test_locally_with_resultstore_results.bazelrc \
```

Comment:
Should this bazelrc be added via the Bazel wrapper?

Reply:
not really (or at least not yet). In all the other builds we have, the options from  test_locally_with_resultstore_results.bazelrc are set separately from the bazel wrapper (the wrapper only sets the "kokoro specific" bits, like the kokoro "Bazel" link, kokoro job invocation details etc.

We can potentially reconsider as a followup.

Reply:
Sounds good. It's not a blocker for this PR.

With the bazel wrapper, the script can do many magical things. The bazelrc can be picked by the input of the bazel report helper or by the platform.

Reply:
Agreed. We need to find good balance between "too much magic" (wrapper has too much automatic stuff in and stops being transparent and predictable) and "not enough magic" (does too little so it doesn't help in practice) though.